contentcuration/contentcuration/frontend/channelEdit/pages/StagingTreePage/index.spec.js

```diff
@@ -109,6 +109,8 @@ const initWrapper = ({ getters = GETTERS, actions = ACTIONS, mutations = MUTATIO
     store,
     stubs: ['MainNavigationDrawer', 'OfflineText'],
   });
+  wrapper.setData({ isLoading: false });
```

Comment:
I would suggest not reaching into internals when testing unless there is no other way. Such dependencies become very hard to track over time and can very easily cause future tests to return values that don't reflect the actual behavior of the view because its value, when mounted, is different. Another thing is that `initWrapper` is meant to prepare the view with as much clean state as possible. Also, renaming `isLoading` internal property to `loading` isn't something that I would expect that needs to be updated in tests. And last but not least, if someone would want to test loading behavior, this would need to be refactored, which is also not what I would expect from a test suite.

Our view communicates with the external world, Vuex, through actions, so I would suggest rather mocking actions that are responsible for the initial data load as resolved, [for example, in `beforeEach` if it's needed for more tests](https://github.com/learningequality/studio/pull/2449/files#diff-1595d9df8cbd766a4b4d4dfd44e82dba7f12c9a7c6c52867fd05cfcfd64a8bfaR170).

Reply:
Updated!